### DISCUSSION TITLE
作为使用者，是否能够上项目的 README 中

### DISCUSSION THREAD

**Discussion by shenxgan** [LagrangeDev/Lagrange.Core#495] [Show and tell]
比如我的 python 学习机器人 https://github.com/shenxgan/qqbot
目前拥有两大功能：
1. 可根据 python 知识点关键字触发相关知识的回复，比如 list 会触发回复 list 的简要介绍
2. 可运行 python 代码，将代码执行结果回复到 qq 群

**Comment by Lilyltt** [comment]
我觉得也许可以,后面会考虑开一个友链区,不过应该会限制仅直接使用core开发的,ob的话还是看看ob那边吧

**Reply by shenxgan** [reply]
嗯，ob那边是哪里？

**Reply by dogdie233** [reply]
> 嗯，ob那边是哪里？

onebot协议文档的 `生态` 页

**Reply by NoirHare** [reply]
你本质是对接 onebot 协议，和 lgr 本身没啥关系

**Comment by Decrabbityyy** [comment]
If your repo based Lagrange.Core,it will more possible
